R/data.R

```diff
@@ -52,3 +52,68 @@
 #' @source Compiled by Alessandro Chiarenza. See item descriptions for details.
 "GTS2012"
 
+#' Early tetrapod data from the Paleobiology Database
+#'
+#' A dataset of tetrapod occurrences ranging from the Carboniferous through to
```

Comment:
Should we make it clear that this is an example dataset here (and that it is a raw download for showing the functionality of package)?